util/network-devp2p/src/discovery.rs

```diff
@@ -555,6 +566,7 @@ impl<'a> Discovery<'a> {
 
 		response.append(&echo_hash);
 		append_expiration(&mut response);
+		response.append(&self.node_record.seq());
```

Comment:
That seems like a breaking change, shouldn't that be reflected in some version field?

Reply:
It is [optional](https://github.com/ethereum/devp2p/blob/master/discv4.md#pong-packet-0x02) and not reflected in the version field. It is a great question though.